client/src/app.tsx

```diff
@@ -137,15 +137,12 @@ export const client = new ApolloClient({
 
 const token = localStorage.getItem(process.env.AUTH_TOKEN);
 
-// Render React app in DOM
-// TODO: fix bad type cast
 ReactDOM.render(
   <Provider store={store}>
-    <ApolloProvider client={client as any}>
-      <ApolloHooksProvider client={client as any}>
-        <Routes />
-      </ApolloHooksProvider>
+    <ApolloProvider client={client}>
+      <Routes />
     </ApolloProvider>
   </Provider>,
-  document.getElementById("application-wrapper")
+  document.getElementById("application-wrapper") ||
```

Comment:
Mounting elements with jest calls this function and returns `Invariant Violation: Target container is not a DOM element.` because `index.html` is not present. Here, all I do is get the `application-wrapper` div if it exists, or return an empty div if it doesn't, which solves the aforementioned issue.

Reply:
:+1: